Source/JavaScriptCore/wasm/WasmAirIRGeneratorBase.h

```diff
@@ -4105,7 +4105,7 @@ auto AirIRGeneratorBase<Derived, ExpressionType>::addI32Popcnt(ExpressionType ar
 {
     result = self().g32();
 
-#if CPU(X86_64) || CPU(ARM64)
+#if CPU(X86_64)
```

Comment:
Can we do similar implementation as what you did in WasmB3IRGenerator.cpp for ARM64?

Reply:
I don't think we should have that since we will remove AirIR generator soon.